src/wp-includes/script-loader.php

```diff
@@ -416,7 +416,7 @@ function wp_default_packages_inline_scripts( $scripts ) {
 						'datetimeAbbreviated' => __( 'M j, Y g:i a' ),
 					),
 					'timezone' => array(
-						'offset' => get_option( 'gmt_offset', 0 ),
+						'offset' => (float) get_option( 'gmt_offset', 0 ),
```

Comment:
What's the difference between `(float)` and `floatval`?

Reply:
Developer preference 😄